### PR TITLE
feat: Configure tools application registry category - EXO-61480 - Meeds-io/MPIs#37

### PR DESCRIPTION
**Pull request by sofyenne**
This change is going to add `External Spaces List Portlet , SpaceMenuPortle and WhoIsOnLinePortlet ` under the Tools application registry category.

### CODE REVIEW

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/application-registry-configuration.xml [hunk 1]
```diff
@@ -250,5 +250,80 @@
...
+                      <field name="description"><string>Who Is OnLine Portlet Portlet</string></field>
+                      <field name="type"><string>portlet</string></field>
+                      <field name="contentId"><string>social-portlet/WhoIsOnLinePortlet</string></field>
+                      <field name="accessPermissions">
+                        <collection type="java.util.ArrayList" item-type="java.lang.String">
+                          <value><string>*:/platform/users</string></value>
+                        </collection>
+                      </field>
+                    </object>
+                  </value>
```
Comment (azayati) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/application-registry-configuration.xml]: separate tags

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/application-registry-configuration.xml (hunk 1 above)
Comment (sofyenne) [reply extension/war/src/main/webapp/WEB-INF/conf/social-extension/portal/application-registry-configuration.xml]: done !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**